src/screen-candidate-detail/viewSelectors.js

```diff
@@ -0,0 +1,11 @@
+import { getCandidate } from '../candidate/redux/candidates';
+
+export const getCandidateSummary = (state, candidateId) => {
```

Comment:
reselect is a good library for creating complex selectors that help bridge the gap between what we store in state and what we display in a view